client/src/pages/HomePage/HomePageForm/HomePageForm.tsx

```diff
@@ -0,0 +1,105 @@
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import CircularProgress from '@mui/material/CircularProgress';
+import { Formik, FormikHelpers } from 'formik';
+import * as Yup from 'yup';
+import useStyles from './useStyles';
+
+interface Props {
+  handleSubmit: (
+    {
+      location,
+      dates,
+    }: {
+      location: string;
+      dates: string;
+    },
+    {
+      setStatus,
+      setSubmitting,
+    }: FormikHelpers<{
+      location: string;
+      dates: string;
+    }>,
+  ) => void;
+}
+
+export default function Login({ handleSubmit }: Props): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Formik
+      initialValues={{
+        location: '',
+        dates: '',
+      }}
+      onSubmit={handleSubmit}
+    >
+      {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
+        <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          <div className={classes.inputLabels}>WHERE</div>
```

Comment:
Please avoid using CAPITALCASE for text, some screen readers interpret this text as abbreviations, you can resort to using a CSS class to transform text instead.
Replace `div` element with MUI `Box` component